schema/query-log-schema-2g.json

```diff
@@ -0,0 +1,164 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "definitions": {
+    "unit": {
+      "type": "object",
+      "properties": {
+        "type": {
+          "type": "string",
+          "enum": [
+            "second",
+            "minute",
+            "hour",
+            "day",
+            "week",
+            "bit",
+            "byte",
+            "rotation",
```

Comment:
Oh I think we've had this conversation before - rotation is used for things like disk speed - 20k rotations / second. Deleting.